Embed News ticker

ChrisJohnRiley opened the pull request:
Display recent news on the `mvsp.dev` site

touzoku commented:
How about considering the option of making it a flat list? It can sometimes be a bit tiring waiting for the ticker to update.

ChrisJohnRiley commented:
> How about considering the option of making it a flat list? It can sometimes be a bit tiring waiting for the ticker to update.

Moved to a static news block (hidden on smaller screens)
Corrected some of the margins

PTAL

touzoku reviewed (approved):
👍
